### PR TITLE
addbasicloginview

### LINKED ISSUE #4: Create basic login view
example: image

### PR DESCRIPTION
**Pull request by przemilymaciej**
Add basic login view with basic CSS.
image

### CODE REVIEW

File: views/loginView.ejs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <div class="header">
+            <h2>Member login</h2>
+        </div>
+        <div class="main">
+            <div class="linput"> <input type="email" placeholder="email"></div>
+            <div class="linput"> <input type="email"placeholder="password"></div>
+            <button class="btn">LOGIN</button>
+            <div class="forgot"><a href="#">Forgot password?</a></div>
+            <div class="newAccount"><a href="#">Create new Account</a></div>
+        </div>
```
Comment (bartlomiejmont) [inline views/loginView.ejs]: wrap this in form field like this: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/forms

Comment (przemilymaciej) [reply views/loginView.ejs]: done

File: views/loginView.ejs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  </head>
+  <body>
+    <div class="container">
+        <div class="header">
+            <h2>Member login</h2>
+        </div>
+        <div class="main">
+          <form method="post">
+            <div class="linput"> <input type="email" placeholder="email"></div>
+            <div class="linput"> <input type="email"placeholder="password"></div>
```
Comment (bartlomiejmont) [inline views/loginView.ejs]: input type password https://www.w3schools.com/html/html_form_input_types.asp

File: views/loginView.ejs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            <button class="btn">LOGIN</button>
```
Comment (bartlomiejmont) [inline views/loginView.ejs]: type="submit"

File: views/loginView.ejs (hunk 2 above)
Comment (przemilymaciej) [reply views/loginView.ejs]: done

File: views/loginView.ejs (hunk 3 above)
Comment (przemilymaciej) [reply views/loginView.ejs]: done

**Review by bartlomiejmont — APPROVED** [review]
ok

**Pull request merged**